app/workers/subject_queue_worker.rb

```diff
@@ -15,6 +15,8 @@ def perform(workflow_id, user=nil, limit=SubjectQueue::DEFAULT_LENGTH)
     else
       load_subjects
     end
+  rescue ActiveRecord::RecordNotFound
```

Comment:
Same thing here -> Wondering here if a user:workflow queue had existed but the workflow was deleted before this event fired via sidekiq. If we should then cleanup any queues that may exist for that workflow?